tests/ci/cdk/README.md

```diff
@@ -46,34 +49,32 @@ To setup or update the CI in your account you will need the following IAM permis
   * ssm:Put*
   * ssm:Update*
   * ssm:List*
+* SecretsManager
+  * secretsmanager:CreateSecret
+  * secretsmanager:PutSecretValue
+  * secretsmanager:DeleteSecret
+  * secretsmanager:GetSecretValue
 
 ### Command
 
+To set up AWS-LC CI, run command:
 ```
-$ ./run-cdk.sh {aws-account-id} {region} {github-repo-owner} {github-repo-name} {ACTION}
+./run-cdk.sh --action deploy-ci --github-access-token 123456
```

Comment:
I'm assuming that `--github-access-token` is something that the user provides? If so, I think it would make it more clear to have them set an environment variable with their token and then use that. Aren't we already using an environment variable somewhere with the user's personal github access token?

```
export GITHUB_ACCESS_TOKEN=...
./run-cdk.sh --action deploy-ci --github-access-token ${GITHUB_ACCESS_TOKEN}
```

Reply:
Previously, to connect AWS CodeBuild with GitHub, personal GitHub access token was used. Now, it's replaced with GitHub OAuth. Besides, this new access token needs `read:package` permission, which is not needed during CI.

Reply:
Added the env variable.